external-authority-sources.md

```diff
@@ -0,0 +1,283 @@
+          }
+        }
+      },
+      {
+        "id": "Smith, Charles",
+        "display": "Smith, Charles",
+        "value": "Smith, Charles",
+        "metadata": {
+            "dc.identifier.orcid": "0000-0003-3681-2038",
+            "dc.identifier.uri": "https://orcid.org/0000-0003-3681-2038",
+            "dc.contributor.other": "University of Mississippi: University, MS, US",
+            "person.familyName": "Smith",
+            "person.givenName": "Charles"
+        },
+        "type": "externalSource",
+        "_links": {
+          "entity": {
+            "href": "https://dspace7-internal.atmire.com/server/api/core/item/6fd90bf5-b84f-47b3-aaec-a55bde3a2a5a"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/orcid/entryValues/0000-0003-3681-2038"
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+### single entry
+**GET /api/integration/externalsources/<:authority-name>/entryValues/<:entry-id>**
+
+It returns the data from one entry in an external source
+
+sample for an external source /api/integration/externalsources/orcid/entryValues/0000-0002-4271-0436 
+```json
+{
+  "id": "Smith, Dean",
+  "display": "Smith, Dean",
+  "value": "Smith, Dean",
+  "metadata": {
+      "dc.identifier.orcid": "0000-0002-4271-0436",
+      "dc.identifier.uri": "https://orcid.org/0000-0002-4271-0436",
+      "dc.contributor.other": "University of Texas Southwestern Medical Center: TX, TX, US",
+      "person.familyName": "Smith",
+      "person.givenName": "Dean"
+  },
+  "type": "externalSource",
+  "_links": {
+    "self": {
+      "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/orcid/entryValues/0000-0002-4271-0436"
+    }
+  }
+}
+```
+
+**POST /api/integration/externalsources/<:authority-name>/entryValues/<:entry-id>/authority**
+
+It creates an authority records from the external source 
+
+sample for an external source /api/integration/externalsources/orcid/entryValues/0000-0002-4271-0436/authority
+```json
+{
+  "id": "Smith, Dean",
+  "display": "Smith, Dean",
+  "value": "Smith, Dean",
+  "metadata": {
+      "dc.identifier.orcid": "0000-0002-4271-0436",
+      "dc.identifier.uri": "https://orcid.org/0000-0002-4271-0436",
+      "dc.contributor.other": "University of Texas Southwestern Medical Center: TX, TX, US",
+      "person.familyName": "Smith",
+      "person.givenName": "Dean"
+  },
+  "type": "authority",
+  "_links": {
+    "self": {
+      "href": "https://dspace7-internal.atmire.com/server/api/integration/authorities/authors/entries/d4b5ca88-9d6d-4a87-b905-fef0f8cae26c"
+    },
+    "externalsource": {
+      "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/orcid/entryValues/0000-0002-4271-0436"
+    }
+  }
+}
+```
+
+**POST /api/integration/externalsources/<:authority-name>/entryValues/<:entry-id>/entity**
```

Comment:
These two options can also support creating the new item. I would prefer the second in that case because:
* The former will be very complicated for the UI
* The https://github.com/DSpace/Rest7Contract/blob/master/workflowitems.md#post-method has to be fixed to support a URI-list for defining the collection anyway (one of the current problems when starting a new submission). So adding the URI of the entry id at the same time and include the metadata would be more straightforward.
* There may be some metadata fields required for the external source which can't be entered using the submission forms (e.g. you'd need to store the source and the ID in that source in dedicated fields so it can be verified later whether there's an entity which matches this external record exactly)